lib/ansible/plugins/httpapi/ftd.py

```diff
@@ -101,10 +108,14 @@ def refresh_token_payload(refresh_token):
         else:
             raise AnsibleConnectionFailure('Username and password are required for login in absence of refresh token')
 
-        dummy, response_data = self.connection.send(
-            self._get_api_token_path(), json.dumps(payload), method=HTTPMethod.POST, headers=BASE_HEADERS
+        url = self._get_api_token_path()
+        self._display(HTTPMethod.POST, 'login', url)
+
+        dummy, response_data = self._send_auth_request(
+            url, json.dumps(payload), method=HTTPMethod.POST, headers=BASE_HEADERS
         )
-        response = self._response_to_json(response_data.getvalue())
+
+        response = self._response_to_json(self._get_response_value(response_data))
```

Comment:
It might also be useful to print login response code to supplement earlier login method debug. white testing this patch I felt login did not go through

Reply:
Thanks, added `status_code` to be printed.